packages/x-components/src/x-modules/queries-preview/components/query-preview.vue

```diff
@@ -0,0 +1,316 @@
+<template>
+  <NoElement v-if="queryPreviewResults && queryPreviewResults.totalResults">
```

Comment:
Should we use the same formula that we used for other components? Like having a list that can be concatenated with another list and finally you can modify the component using the slot:

![image](https://user-images.githubusercontent.com/6597815/188382239-88b1f1fb-18f4-453e-a962-7a74632e1a78.png)

I know that this is already working, but im not sure if we want to have the same way of concatenating things that are going to render results.